components/com_finder/tmpl/search/default_result.php

```diff
@@ -115,7 +115,13 @@
                     <?php endforeach; ?>
                     <?php if (count($taxonomy_text)) : ?>
                         <li class="result__taxonomy-item result__taxonomy--<?php echo $type; ?>">
-                            <span><?php echo Text::_(LanguageHelper::branchSingular($type)); ?>:</span> <?php echo implode(',', $taxonomy_text); ?>
+                            <span><?php echo Text::_(LanguageHelper::branchSingular($type)); ?>:</span>
+                            <?php $text = implode(',', $taxonomy_text); ?>
+                            <?php if ($type === 'Type') : ?>
+                                <?php echo Text::_($text); ?>
```

Comment:
```suggestion
                                <?php $key = LanguageHelper::branchSingular($text); ?>
                                <?php echo $lang->hasKey($key) ? Text::_($key) : $text; ?>
```
When I do it like this, it works here.